doc/modules/mod_event_pusher_rabbit.md

```diff
@@ -24,9 +24,13 @@ bare jid (`user@domain`) and configurable topic e.g `alice@localhost.private_mes
 * **amqp_port** (integer, default: `5672`) - Defines RabbitMQ server AMQP port;
 * **amqp_username** (string, default: `<<"guest">>`) - Defines RabbitMQ server username;
 * **amqp_password** (string, default: `<<"guest">>`) - Defines RabbitMQ server password;
+* **confirms_enabled** (boolean, default: `false`) - Enables/disables one-to-one publishers confirms;
 * **presence_exchange** (string, default: `<<"presence">>`) - Defines RabbitMQ presence exchange name;
+* **presence_exchange_type** (string, default: `<<"topic">>`) - Defines RabbitMQ presence exchange type;
```

Comment:
Hmm this configuration seems to be a little bit to complex. Why not simplifying it a bit and go for a format like:

```erlang
{Exchange :: atom(), {Name :: string(), Type :: atom()}}
%% or
{Exchange :: atom(), Name :: string()} %% then we have the default type
```

Reply:
For example:

```erlang
{presence_exchange, <<"presence">>, direct}
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```erlang
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```erlang
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```erlang
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```erlang
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```

Reply:
@mentels but what if one want to change only exchange name or type? What do you think about that?

```
{presence_exchange, [{name, <<"presence">>, {type, <<"direct">>}]
```